src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/rollup/RollupMapperService.kt

```diff
@@ -197,7 +198,8 @@ class RollupMapperService(
     private fun isFieldInMappings(fieldName: String, mappings: Map<*, *>): Boolean {
         var currMap = mappings
         fieldName.split(".").forEach { field ->
-            val nextMap = (currMap[PROPERTIES] as Map<*, *>?)?.get(field) ?: return false
+            // Looks for a properties field to continue searching or if there is none checks if there is a subfield
+            val nextMap = (currMap[PROPERTIES] as Map<*, *>? ?: currMap[IndexUtils.FIELDS] as Map<*, *>?)?.get(field) ?: return false
```

Comment:
NP: Can either import `IndexUtils.FIELDS` like `PROPERTIES` (so it's just referred to as `FIELDS`) or vice-versa. Just as long as they're consistent.